src/rules/getter_return.rs

```diff
@@ -386,134 +401,129 @@ const obj = {
 
   #[test]
   fn getter_return_invalid() {
-    fn msg1(getter_name: &str) -> String {
-      format!("Expected to return a value in '{}'.", getter_name)
-    }
-
-    fn msg2(getter_name: &str) -> String {
-      format!("Expected '{}' to always return a value.", getter_name)
-    }
-
     assert_lint_err! {
       GetterReturn,
 
       // object getter
       "const foo = { get getter() {} };": [
         {
           col: 14,
-          message: msg1("getter"),
-          hint: "Return a value from the getter function",
+          message: GetterReturnMessage::Expected("getter".to_string()),
+          hint: GetterReturnHint::Return,
```

Comment:
`.to_string()` may look noisy here. I tried to remove it by some macro, but I couldn't. Probably it requres proc macro.

Reply:
The solution looks great, but those `to_string()` calls are noisy. @magurotuna maybe we could use `&str` instead with appropriate lifetime?

Reply:
sure, I'll give it a shot.

Reply:
hmmm... I'm doing my best but using `&str` in GetterRetrunMessage seems extremely difficult for me :/ (How I tried is here: 7759d4b810737ef3086f036e0f154fc0e307eb73)
I'll try another solution using a declarative macro.